### DISCUSSION TITLE
inquirer repl over SSH

### DISCUSSION THREAD

**Discussion by stefaanMLB** [SBoudrias/Inquirer.js#1337] [Q&A]
Is it possible to embed an ssh client into a nodejs application ?
For large applications, it would be great for ops people to be able to SSH into the application and have REPL that helps then do whatever tasks they have to do.
Any ideas if/how this can be done ?

**Comment by SBoudrias** [comment]
There might be some Node.js clients, but maybe the easier way would be to [spawn](https://nodejs.org/api/child_process.html) any ssh client available on the machine from node and inherit the stdin/stdout/stdio
